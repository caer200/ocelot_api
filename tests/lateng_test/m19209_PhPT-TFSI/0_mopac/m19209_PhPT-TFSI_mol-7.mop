CHARGES

 
C	2.540561 +1 7.463345 +1 19.135073 +1
S	2.303249 +1 9.106497 +1 19.924453 +1
F	3.410565 +1 7.511591 +1 18.147335 +1
F	1.406776 +1 6.996626 +1 18.644671 +1
F	2.989959 +1 6.612165 +1 20.044836 +1
N	1.613801 +1 9.963093 +1 18.792830 +1
O	1.326704 +1 8.857958 +1 20.946024 +1
O	3.614188 +1 9.514766 +1 20.317733 +1
S	2.422403 +1 10.739068 +1 17.682968 +1
C	2.595823 +1 12.427192 +1 18.374976 +1
O	3.766449 +1 10.313920 +1 17.455244 +1
O	1.571815 +1 10.916812 +1 16.544817 +1
F	3.283943 +1 12.417745 +1 19.506898 +1
F	3.241708 +1 13.192460 +1 17.502003 +1
F	1.410344 +1 12.968486 +1 18.615065 +1
