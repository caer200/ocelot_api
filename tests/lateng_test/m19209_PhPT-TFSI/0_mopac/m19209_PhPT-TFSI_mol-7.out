 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 22:29:56 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          2.54056100  *   7.46334500  *  19.13507300  *
     2       S          2.30324900  *   9.10649700  *  19.92445300  *
     3       F          3.41056500  *   7.51159100  *  18.14733500  *
     4       F          1.40677600  *   6.99662600  *  18.64467100  *
     5       F          2.98995900  *   6.61216500  *  20.04483600  *
     6       N          1.61380100  *   9.96309300  *  18.79283000  *
     7       O          1.32670400  *   8.85795800  *  20.94602400  *
     8       O          3.61418800  *   9.51476600  *  20.31773300  *
     9       S          2.42240300  *  10.73906800  *  17.68296800  *
    10       C          2.59582300  *  12.42719200  *  18.37497600  *
    11       O          3.76644900  *  10.31392000  *  17.45524400  *
    12       O          1.57181500  *  10.91681200  *  16.54481700  *
    13       F          3.28394300  *  12.41774500  *  19.50689800  *
    14       F          3.24170800  *  13.19246000  *  17.50200300  *
    15       F          1.41034400  *  12.96848600  *  18.61506500  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         C          2.5406      7.4633     19.1351
     2         S          2.3032      9.1065     19.9245
     3         F          3.4106      7.5116     18.1473
     4         F          1.4068      6.9966     18.6447
     5         F          2.9900      6.6122     20.0448
     6         N          1.6138      9.9631     18.7928
     7         O          1.3267      8.8580     20.9460
     8         O          3.6142      9.5148     20.3177
     9         S          2.4224     10.7391     17.6830
    10         C          2.5958     12.4272     18.3750
    11         O          3.7664     10.3139     17.4552
    12         O          1.5718     10.9168     16.5448
    13         F          3.2839     12.4177     19.5069
    14         F          3.2417     13.1925     17.5020
    15         F          1.4103     12.9685     18.6151

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C2 N O4 S2 F6  =    15 atoms



      MOLECULAR POINT GROUP   :   C2  

   TOPOGRAPHY OF SYSTEM

   ATOM No.    LABEL   Atoms connected to this atom
      1          C      2      3      4      5
      2          S      7      8      6      1
      3          F      1
      4          F      1
      5          F      1
      6          N      2      9
      7          O      2
      8          O      2
      9          S     11     12      6     10
     10          C      9     13     14     15
     11          O      9
     12          O      9
     13          F     10
     14          F     10
     15          F     10

     Ion Atom No.  Type    Charge
 
    1       6      N       +1

          COMPUTED CHARGE ON SYSTEM: +1

          COMPUTED CHARGE ON SYSTEM: +1

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
